src/main/java/edu/cornell/kfs/fp/businessobject/TransactionDTO.java

```diff
@@ -0,0 +1,106 @@
+package edu.cornell.kfs.fp.businessobject;
+
+import java.io.Serializable;
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+public class TransactionDTO implements Serializable {
```

Comment:
This is kind of a generic name - maybe you can come up with a more specific one that indicates exactly what this is?